src/main/java/hu/oe/nik/szfmv/automatedcar/xml/XmlParser.java

```diff
@@ -0,0 +1,108 @@
+package hu.oe.nik.szfmv.automatedcar.xml;
```

Comment:
Miért nem a model/utilityben van? Szerintem fölösleges neki egy package amibe semmi más sem fog kerülni

Reply:
Szerintem logikailag nem tartozik a modellhez, XML feldolgozást végez, ami modelleket ad vissza. Attól, hogy modell csapat vagyunk, nem feltétlen kell a modell packageben dolgoznunk. A modellt szerintem tisztán arra kellene meghagyni, ami szigorúan az objektum felépítését tartalmazza, illetve olyan funkciókat, amiket a modellnek "önmagának" kell tudnia. Szerintem ez már nem ilyen funkció, így jó helyen van. Ha mondjuk kiegészülne egy mentés funkcióval, amivel az aktuális világ állapotát mentenénk el xml-be, akkor azt is ebben az osztályban kellene implementálni és itt már jön is az első bibi, rossz az elnevezés! Lehetne XmlService (bár ez inkább J2EE-s fogalom), XmlUtil, mivel akkor már nem csak beolvasással foglalkozik.

Sok mindent máshogy látunk, én sok mindent refaktorálnék is, bocsi, de nagyon "belém égett" a J2EE-s logikai, ami szerintem baromi jó, de majd annak idején ezeket megbeszéljük. :)

Reply:
Én azt javaslom, hogy ne blokkoljuk most a GUI csapatot, majd refaktorálunk, gyakorlatilag csak importokat kell majd cserélni.

Reply:
Meggyőztél, legyen akkor az XML package